src/blocks/modal.css

```diff
@@ -11,7 +11,9 @@
   display: flex;
   justify-content: center;
   align-items: center;
+  z-index: 2;
 }
+
```

Comment:
* Please, correct the `font` for the `delete` and `avatar` modals. It should be `font-family: Poppins, Arial, Helvetica, "sans-serif";`. You can add it into `modal.css` on line 2, so that all modals could have it by default